### ISSUE TITLE
Sets MFD brightness on launch

### ISSUE THREAD

**Issue by wentam** [performantp/x52MFD#2]
Running x52MFD sets the MFD brightness on launch. This can be annoying when using another program/library to set the MFD brightness.

**Comment by performantp** [comment]
Agreed, it should be read first - I don't remember if the library allowed for that.
